code/modules/mob/living/carbon/xenomorph/castes/queen/queen.dm

```diff
@@ -38,6 +39,9 @@
 	hive.RegisterSignal(src, COMSIG_HIVE_XENO_DEATH, /datum/hive_status.proc/on_queen_death)
 	playsound(loc, 'sound/voice/alien_queen_command.ogg', 75, 0)
 
+	// Upon ascension to queen, hive construction permissions default to forbidden.
+	hive.building_allowed = XENO_BUILDING_FORBIDDEN
```

Comment:
yeah no, that's a bad idea

Reply:
Alright, here's my logic. I and other queens will 100% forget that this option exists. Defaulting to "unrestricted" building of silos and turrets invalidates the entire reason why this PR exists, to prevent bald hivelords from fucking over the entire hive by siloing and turreting outside of FOB. Then we need to AHelp and hope that whatever admins are on, if they are, dont respond with "skill issue" and send 800+ points down the drain.

It's like CM with building permissions. If someone wants to silo or turret while the setting is NONE / Forbidden, just ask and they'll toggle it on.

Reply:
Ya know what, i'll refactor this into a button. The Psychic Whisper icon is currently free,

Reply:
BM has a point saying that it should default to unrestricted (unless he is saying the code is bad, idk). Having a bad hivelord or shrike wasting all the psy points is bad, but having a bald queen not building a silo and the hive collapsing is worse. You can't assume that all queens are competent. Your logic is also pretty poor, making a PR giving the queen the responsibility for all psy point usage and then saying that the queen can't remember clicking a button.

Reply:
Fine, I'll give the shrike the ability to ignore queen restrictions. I find that shrikes are far less likely to grief and burn points and far more likely to ask in hivemind before building.

Reply:
Fine